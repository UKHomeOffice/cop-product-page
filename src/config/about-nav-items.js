export const navItems = [
  { name: "About COP", href: "/about/" },
  {
    name: "How COP helps you",
    href: "/about/benefits/",
  },
  { name: "Roadmap", href: "/about/roadmap/" },
  { name: "Security", href: "/about/security/" },
];
