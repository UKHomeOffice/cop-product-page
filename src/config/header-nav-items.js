export const navItems = [
  { name: "Home", href: "/" },
  { name: "About", href: "/about/" },
  { name: "Help", href: "/help/accessing/" },
  { name: "Support", href: "/support/" },
  { name: "Sign-in", href: "https://www.cop.homeoffice.gov.uk/" }
];