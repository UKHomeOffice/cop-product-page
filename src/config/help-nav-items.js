export const navItems = [
  {
    name: "Accessing your services", href: "/help/accessing/"
  },
  {
    name: "Operational Activity Reporting", href: "/help/oar/",
    subItems: [{ name: "Overview", href: "/help/oar/" },
      { name: "Before you begin", href: "/help/oar/oar-before-you-begin-your-oar/" },
      { name: "Enter shift details and time", href: "/help/oar/oar-entering-shift-details-and-time/" },
      { name: "Understand activity categories", href: "/help/oar/oar-understanding-activity-categories/" },
      { name: "Detection/search activities", href: "/help/oar/oar-detection-search-activities/" },
      { name: "Check your OAR submission", href: "/help/oar/oar-checking-your-oar-submission/" },
    ]
  },
  {
    name: "Mandatory Declarations", href: "/help/mandec/",
    subItems: [{ name: "Overview", href: "/help/mandec/" },
      { name: "Add your line manager", href: "/help/mandec/mandec-adding-your-line-manager/" },
      { name: "If you have something to declare", href: "/help/mandec/mandec-something-to-declare/" },
      { name: "What Line Managers need to do", href: "/help/mandec/mandec-what-line-managers-need-to-do/" }
    ]
  }
];
