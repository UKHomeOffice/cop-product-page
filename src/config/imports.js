import AccordionRoadmap from "../components/accordion-roadmap";
import Accordion from "../components/accordion";
import BoxIcon from "../components/box-icon";
import BreakLine from "../components/Break-line";
import BulletList from "../components/bullet-list";
import { Button } from "../components/button";
import ExtraLargeTitle from "../components/extra-large-title";
import Hr from "../components/hr";
import HyperLink from "../components/hyperlink";
import Image from "../components/image";
import LargeParagraph from "../components/large-paragraph";
import LargeTitle from "../components/large-title";
import Logo from "../components/logo";
import MediumTitle from "../components/MediumTitle";
import Paragraph from "../components/paragraph";
import RoadMapBlock from "../components/roadmap-block";
import RoadMapLabel from "../components/roadmap-labels";
import RoadMapBlockContainer from "../components/RoadMapBlockContainer";
import SmallTitle from "../components/SmallTitle";
import Video from "../components/video";
import StatContent from "../components/stats-content";
import layout from "../components/layout";
import Panel from "../components/panel";
import ImageNewWindow from "../components/image-new-window";

export {
  Accordion,
  AccordionRoadmap,
  BoxIcon,
  BreakLine,
  BulletList,
  Button,
  ExtraLargeTitle,
  Hr,
  HyperLink,
  Image,
  LargeParagraph,
  LargeTitle,
  Logo,
  MediumTitle,
  Paragraph,
  RoadMapBlock,
  RoadMapLabel,
  RoadMapBlockContainer,
  SmallTitle,
  Video,
  StatContent,
  layout,
  Panel,
  ImageNewWindow,
};
