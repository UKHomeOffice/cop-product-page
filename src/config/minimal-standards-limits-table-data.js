import React, { useEffect } from "react";
import MinimalStandardLimitsTable from "../components/minimal-standard-limits-table"

const tableContent = {
    PriorityA: [
        {
            column1: '-',
            column2: '-',
            column3: 'Frontline to record in…'
        },
        {
            column1: '<strong>COVID-19</strong>',
            column2: 'All Incidents, where items/ people detection activity relates to COVID-19.',
            column3: 'Seizure file/ Immigration Case file'
        },
        {
            column1: '<strong>National Security at the Border</strong> (including CBRN - chemical, biological, radiological and nuclear)',
            column2: '<p>All incidents</p>  <p>All positive incidents at level 3 as defined by the Cyclamen operating procedures (COPs).</p> <p>ALL CASES relating to COVID-19</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Clandestine Entry</strong>',
            column2: 'Any clandestine you find at the UK border including juxtaposed controls. ALL CASES relating to COVID-19',
            column3: 'Immigration case file'
        },
        {
            column1: '<strong>Class A</strong>',
            column2: 'All seizures of 50g and above.',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Strategic exports</strong>',
            column2: 'All cases in which you suspect an attempt has been made to circumvent prohibitions, sanctions or restrictions to acquire certain goods that are subject to UN, EU or Organisation for Security and Cooperation in Europe (OSCE) controls or orders.',
            column3: 'Seizure file'
        },
        {
            column1: '<Strong>Cash</strong>',
            column2: '<p>All detections of cash linked to terrorism, whatever the amount.</p> <p>All cash seizures of £20,000 and over.</p> <p>All cash detections where there is an unusual concealment or method.</p> <p>All cash detections involving abuse or misuse of occupation or employment, for example government official, or airport or port worker.</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Firearms – Lethal.</strong> Including component parts',
            column2: 'All seizures',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Modern Slavery and Human trafficking</strong> – defined as ‘the recruitment, transportation, transfer, harbouring or receipt of persons by means of the threat or use of force or other forms of coercion, abduction, fraud, abuse of power’.',
            column2: '<p>Any cases in which there is evidence, suspicion, or reasonably held belief that the person may be involved in, or the subject of, human trafficking. This includes documents linked to human trafficking found during examinations of baggage and/or freight (for example: forged or genuine passports and correspondence).</p> <p>ALL CASES relating to COVID-19</p>',
            column3: 'Immigration case file'
        },
        {
            column1: '<strong>Child Sexual Abuse and Exploitation</strong> – Section 55 of the Borders, Citizenship and Immigration Act 2009 requires staff to be aware of the need to safeguard and promote the welfare of children (to be alert to the indicators of child neglect and/or abuse).',
            column2: '<p>Any cases in which there is evidence, suspicion, or reasonably held belief that the person may be at risk under Section 55 of the Borders, Citizenship and Immigration Act 2009. This includes documents linked to child protection found during examinations of baggage and/or freight.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Immigration case file'
        },
    ],
    PriorityB: [
        {
            column1: '-',
            column2: '-',
            column3: 'Frontline to record in…'
        },
        {
            column1: '<strong>Intellectual property rights</strong>',
            column2: '<p>All significant cases, or what could be deemed to be commercial quantities, of counterfeit goods imported into the UK.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>International Trade</strong>',
            column2: '<p>All Detections including Non-compliance identified/ product safety.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Medicines and Medical Services</strong>',
            column2: '<p>All seizures of 100 tabs and/or 250g and above.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Cigarettes</strong>',
            column2: '<p>All seizures of 10,000 cigarettes and above, or any unusual concealments or method.</p><p>All cigarette detections involving abuse or misuse of occupation or employment – for example government official or airport or port worker.</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Hand-rolling tobacco</strong>',
            column2: '<p>All seizures of 10kg and above, or any unusual concealments or method.</p> <p>All tobacco detections involving abuse or misuse of occupation or employment – for example government official or airport or port worker.</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Other tobacco products</strong> – for example fruit tobacco or pipe tobacco',
            column2: '<p>All seizures of 10kg and above, or any unusual concealments or method. This includes:</p> <ul class="govuk-list govuk-list--bullet"><li>cigarette and tobacco-making equipment</li><li>tobacco shredders</li><li> counterfeit cigarette and tobacco packaging</li><li>unprocessed and/or raw tobacco</li></ul>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Common travel area (CTA) abuse</strong>',
            column2: '<p>Any arrests, interceptions, detections and referrals to the criminal financial investigation teams when you suspect abuse of the CTA.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Immigration case file'
        },
        {
            column1: '<p style="color: white"><strong>Immigration identity crime:</strong> including:</p><ul style="color: white" class="govuk-list govuk-list--bullet"><li> inadequately documented arrivals (IDA)</li><li> transit without visa (TWOV) abuse</li><li>visa abuse</li></ul>',
            column2: '<p>Any cases in which there is evidence or suspicion of the involvement of organised criminality, widespread document abuse or persistent criminal behaviour or, due to the severity of the alleged offence, prosecution is deemed to be the most appropriate course of action.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Immigration case file'
        },
        {
            column1: '<strong>Knives and offensive weapons</strong>',
            column2: 'All seizures of gravity knives, flick knives, knuckle dusters, swords, telescopic batons, pepper sprays and laser products.',
            column3: 'Seizure file'
        },
    ],
    PriorityC: [
        {
            column1: '-',
            column2: '-',
            column3: 'Frontline to record in…'
        },
        {
            column1: '<strong>Alcohol</strong>',
            column2: 'All seizures of 5,000 litres and above.',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Immigration Rules Abuse</strong>',
            column2: '<p>Any cases in which there is evidence or suspicion of the involvement of organised criminality, widespread document abuse or persistent criminal behaviour or, due to the severity of the alleged offence, prosecution is deemed to be the most appropriate course of action.</p><p>ALL CASES relating to COVID-19</p>',
            column3: 'Immigration case file'
        },
        {
            column1: '<strong>Other Class B & C drugs.</strong> including: amphetamines, hallucinogens, and anabolic steroids',
            column2: 'All seizures of 100 tabs and/or 250g and above.',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>CITES</strong> (Convention on International Trade in Endangered Species)',
            column2: '<p>All significant or commercial seizures of traditional Chinese medicines</p><p>All significant or commercial seizures of timber</p><p>All seizures of live animals</p><p>All commercial seizures of dead animals and animal derivatives</p><p>All commercial seizures of plants</p>',
            column3: '&nbsp;'
        },
        {
            column1: '<strong>New psychoactive substances</strong> – also known as legal highs.',
            column2: 'Any cases in which, following detection and subsequent laboratory testing, the substances are found to contain components that are subject to control under the 1961 UN Single Convention on Narcotic Drugs, the 1971 UN Convention on Psychotropic Substances and/or the Misuse of Drugs Act 1971.',
            column3: 'Seizure file'
        },
    ],
    PriorityD: [
        {
            column1: '-',
            column2: '-',
            column3: 'Frontline to record in…'
        },
        {
            column1: '<strong>MTIC (Missing Trader Intra-Community Fraud)</strong>',
            column2: 'All detections. Under review',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Firearms</strong> – Non-Lethal. Including CS sprays and electronic stun guns',
            column2: 'All seizures.',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Products of animal origin (POAO)</strong>',
            column2: '<p>Meat:</p> <ul class="govuk-list govuk-list--bullet"><li>Bushmeat weighing more than 10kg</li><li>Other meat products weighing more than 20kg</li></ul>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Cutting Agent and Precursors</strong>',
            column2: 'All events. Under Review.',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Cannabis</strong>',
            column2: 'All seizures of 5kg and above.',
            column3: 'Seizure file'
        },
    ],
    PriorityE: [
        {
            column1: '-',
            column2: '-',
            column3: 'Frontline to record in…'
        },
        {
            column1: '<strong>Plant Health</strong>',
            column2: '10kgs or more. Under review',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Indecent and obscene material</strong>',
            column2: '<p>All paedophile material including indicative material.</p> <p>Obscene material (rape, bestiality, torture) but not material involving consenting adults.</p><p>Snuff movies (killing of adults and children).</p><p>Commercial quantities larger than what could be deemed for personal use (25-50 tapes), or more than one of a specific tape or DVD.</p>',
            column3: 'Seizure file'
        },
        {
            column1: '<strong>Hydrocarbon Oils</strong>',
            column2: 'All seizures of 5,000 litres and above. Under review',
            column3: 'Seizure file'
        },
    ],
}

export const MinimalStandardLimitsTables = [
    {
        id: 'priorityA',
        header: "Priority A",
        body: <MinimalStandardLimitsTable table={tableContent.PriorityA} />
    },
    {
        id: 'priorityB',
        header: "Priority B",
        body: <MinimalStandardLimitsTable table={tableContent.PriorityB} />
    },
    {
        id: 'priorityC',
        header: "Priority C",
        body: <MinimalStandardLimitsTable table={tableContent.PriorityC} />
    },
    {
        id: 'priorityD',
        header: "Priority D",
        body: <MinimalStandardLimitsTable table={tableContent.PriorityD} />
    },
    {
        id: 'priorityE',
        header: "Priority E",
        body: <MinimalStandardLimitsTable table={tableContent.PriorityE} />
    }
];