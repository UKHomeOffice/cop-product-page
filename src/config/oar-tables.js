import React, { useEffect } from "react";
import OarTable from "../components/oar-table";

const tableContent = {
    transportModes: [
        { rowHeader: 'Air Freight', rowContent: 'Commercial air freight' },
        { rowHeader: 'Air Passenger', rowContent: 'Airlines carrying passengers' },
        { rowHeader: 'Commercial Maritime', rowContent: 'Work with freight-carrying large ships e.g. container ships etc.' },
        { rowHeader: 'Container', rowContent: 'Containers either accompanied or unaccompanied (excluding RoRo see below)' },
        { rowHeader: 'Cruise', rowContent: 'Cruise ship passengers and crew' },
        { rowHeader: 'Fast Parcels', rowContent: '&nbsp;' },
        { rowHeader: 'General Aviation', rowContent: 'Work with private or charter aircraft including drones etc. at small airports and other locations' },
        { rowHeader: 'General Maritime', rowContent: 'Generally carried out with private or small marine craft e.g. at marinas and small ports' },
        { rowHeader: 'International Trade', rowContent: 'Search activities conducted to check that goods comply with trade rules and restrictions and have appropriate documentation, including an accurate description of the goods' },
        { rowHeader: 'Post', rowContent: '&nbsp;' },
        { rowHeader: 'Rail Freight', rowContent: 'Commercial freight carried on trains' },
        { rowHeader: 'Rail Passenger', rowContent: 'Eurostar and other commercial train operators' },
        { rowHeader: 'RoRo Freight', rowContent: 'Commercial freight handling vehicles handled by roll on roll off ferries. Please make it clear whether this applies to accompanied or unaccompanied freight (towed from RoRo to Port)' },
        { rowHeader: 'RoRo Tourist', rowContent: 'Private vehicles and foot passengers handled by roll on roll off ferries' }
    ],
    PCPImmigrationActivities: [
        { rowHeader: 'Record work arising from Primary Control Point (PCP) and Immigration procedures', rowContent: 'Guidelines for recording PCP and Immigration Activities'},
        { rowHeader: 'General Primary Control Point (PCP)', rowContent: '<p>To include all activity involving processing of EEA and Non EEA nationals up to the point of it becoming a case and being entered onto CID all subsequent work is entered into casework (see activity below).</p> <ul class="govuk-list govuk-list--bullet"><li>PCP at airports and seaports.</li> <li>It may also include CTA work activity, the processing of crew, work on cruise liners, by the duty officer at small ports, by teams working at small airfields, on freight or any other area where EEA and Non EEA nationals are cleared.</li></ul> e-Gates - This is to include all officers involved in the processing of passengers through e-Passport Gates.' },
        { rowHeader: 'Immigration Casework', rowContent: '<p>Total hours spent on casework types including:</p><strong>LTE</strong><br><p>Cases recorded on CID that result in LTE (Leave to Enter).</p><strong>TA/DET</strong><br><p>Cases that remain unresolved and result in Temporary Admission (TA) or Detention pending further enquiries.</p><strong>RLE</strong><br><p>Non Visa Cases - Non Visa cases that result in Refusal of Leave to Enter (RLE).</p><strong>RLE Visa Case</strong><br><p>Visa cases that result in RLE.</p><strong>Other Case-Working Duties</strong><br><p>The total number of hours staff are deployed to other case-working duties. This will include any casework Border Force Higher Officers (BFHOs), coordinators or any other officers coordinating, continuing or concluding casework not accounted for in the previous casework sections (for example cases that have been bought back from Temporary Admission or detention to be concluded, and work relating to absconders and removal work activity that is not covered by any of the other casework sections.)</p>'},
        { rowHeader: 'Other Immigration', rowContent: '<p>Deportation Orders, Medical Emergencies.</p><p><strong>Record actual examinations searches and results under ‘Detection/Search’ activities.</strong></p>' },
        { rowHeader: 'National Security (NS) formerly ‘Counter Terrorism’ referrals', rowContent: 'May include passengers or contents of freight, fast parcels and post.' },
        { rowHeader: 'Safeguarding', rowContent: 'Including safeguarding patrols.' },
        { rowHeader: 'Forgery', rowContent: '<p>Further examination of potential forgery cases and use of specialist forgery detection equipment where appropriate.</p><p>Subsequent actions taken with passengers, documents etc. where forgery is detected.</p>' },
        { rowHeader: 'Asylum casework', rowContent: 'Casework for Asylum seekers.' },
        { rowHeader: 'COVID Compliance Checks', rowContent: 'Amount of time spent conducting spot and other checks on people entering the UK to ensure they have completed the necessary declarations and to process any subsequent reports and casework' },
    ],
    customActivities: [
        { rowHeader: 'Custom Activities', rowContent: 'Guidelines for recording Customs Activities' },
        { rowHeader: 'Channel', rowContent: '<p>Also known as ‘Secondary’.</p><p>Include Stop and Question (S&Q) and BDI/profiling</p><p><strong>Record examinations searches and results under ‘Detection/Search’ activities.</strong></p><p>Anti-smuggling checks in the Green Channel (“Nothing to Declare”) and for taking any required further action in line with current customs operational guidance such as making any necessary referrals for civil penalties or for further investigation to HMRC/NCA and making any necessary arrests.</p><p><strong>Record actual search time and results (positive and negative) under ‘Detection/Search’ activities.</strong></p><p>Any resulting activity of recording and processing a positive result (e.g. after a commodity seizure) should be recorded using the appropriate activities available in OAR see ‘Other Activities’ below.</p>' },
        { rowHeader: 'International trade checks', rowContent: '<p>Regulatory international trade controls on goods entering or leaving the EU via the UK to ensure compliance with international, EU and domestic regulatory requirements.</p><p>Checking and endorsement of all international trade documentation requiring a customs stamp at the border to evidence import and export movements and compliance with any relevant customs clearance legal requirements.</p><p>Any other compliance checks to ensure that operators are complying with the conditions of relevant customs procedures governing the movement of cargo within the confines of a port / airport, including movements to temporary storage locations.</p>' },
        { rowHeader: 'Other customs duties', rowContent: '<p>All other customs procedures and paperwork conducted outside of Channel and not included in International Trade Checks as above.</p><p>Investigation of collusion in customs fraud abetted by staff working for port or carriers</p><p><strong>Record examination search time and results under ‘Detection/Search’ activities</strong></p>' },
        { rowHeader: 'Declared goods', rowContent: 'Assess and collect at the border any taxes, duties or other charges liable to be paid on goods from passengers and crew that have declared goods in the Red Channel or at a Red Point facility (“Goods to Declare”).' },
        { rowHeader: 'VAT returns', rowContent: 'Checking and endorsement of VAT refund claims.' },
        { rowHeader: 'Firearms management', rowContent: '<p>Handling weapons for Air Marshals etc..</p><p>All ‘Make Safe’ procedures when clearing declared firearms or with seized firearms.</p><p><strong>Search and detection time where firearms are targetted or found are recorded as ‘Detection/Search’ Activities.</strong></p>' },
    ],
    searchDetectionActivities: [
        { rowHeader: 'Record Detection/Search Activities against each target type – includes all search and detection for Clandestine Immigrants, Victims of Modern Slavery etc. as well as all types of Commodities', rowContent: 'Guidelines for recording Search & Detection Activities'},
        { rowHeader: 'Self-selection', rowContent: 'For example as a result of Stop and Question (S&Q), observation, dog detection etc. including self selected baggage under S159 – (Power to Examine Goods (Passenger Baggage)) searches '},
        { rowHeader: 'A category target', rowContent: '&nbsp;'},
        { rowHeader: 'B category target', rowContent: '&nbsp;'},
        { rowHeader: 'C category target', rowContent: '&nbsp;'},
        { rowHeader: 'Intelligence and third party referrals', rowContent: 'Including airport security, ship security officers and other agencies'},
        { rowHeader: 'Primary Control Point (PCP) referral', rowContent: '&nbsp;'},
        { rowHeader: 'Scanner referral', rowContent: '&nbsp;'},
        { rowHeader: 'Structured Assessment Border Risk (SABR)', rowContent: '&nbsp;'},
    ],
    cyclamenActivities: [
        { rowHeader: 'Cyclamen', rowContent: '&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;'},
    ],
    OtherActivities: [
        { rowHeader: 'Other Activities', rowContent: '&nbsp;'},
        { rowHeader: 'Crew Exercise', rowContent: '&nbsp;'},
        { rowHeader: 'Custody Duties', rowContent: '&nbsp;'},
        { rowHeader: 'Detention management', rowContent: '&nbsp;'},
        { rowHeader: 'GA/GM Monitoring (contact visit)', rowContent: '&nbsp;'},
        { rowHeader: 'GA/GM Monitoring (non-contact visit)', rowContent: '&nbsp;'},
        { rowHeader: 'Health & Safety/Welfare', rowContent: '&nbsp;'},
        { rowHeader: 'Intelligence collection and reporting', rowContent: '&nbsp;'},
        { rowHeader: 'Management meetings', rowContent: '&nbsp;'},
        { rowHeader: 'Other non-operational', rowContent: '<ul class="govuk-list govuk-list--bullet"><li>Union functions</li><li>Other Facility time (Home Office Networks and Groups)</li><li>Administrative functions carried out by officers due to the absence of administrative staff.</li><li>Assurance Meetings</li></ul>'},
        { rowHeader: 'Other operational', rowContent: '<ul class="govuk-list govuk-list--bullet"><li>Reporting Duties (OAR etc.)</li><li>Duty Office</li><li>Management Functions (HO Activities)</li><li>Carriers Liability (CL)</li><li>Overtime Checking</li><li>Information Technology (IT)</li><li>Security</li><li>Secure ID Functions (Secure Identity)</li><li>Equality & Diversity</li><li>Investors in People</li><li>Data Quality</li></ul>'},
        { rowHeader: 'Queen\'s warehouse', rowContent: '&nbsp;'},
        { rowHeader: 'Training', rowContent: '&nbsp;'},
        { rowHeader: 'Travel time - offsite', rowContent: '<p>Nationally (Except Heathrow):</p><ul class="govuk-list govuk-list--bullet"><li>Includes all travel time once away from main site</li></ul> <p>Heathrow:</p><ul class="govuk-list govuk-list--bullet"><li>Tarmac: Any time spent landside by Tarmac teams</li></ul><p>Freight: More than 30 minutes spent travelling by Freight teams</p>'},
        { rowHeader: 'Travel time - onsite', rowContent: '&nbsp;'},
        { rowHeader: 'VIP Movements', rowContent: '&nbsp;'},
    ],
}

export const OarTables = [
    {
        id: 'transportModes',
        header: "Transport Modes",
        body: <OarTable table={tableContent.transportModes}/>
    },
    {
        id: 'pcpImmigrationActivities',
        header: "PCP & Immigration Activities",
        body: <OarTable table={tableContent.PCPImmigrationActivities}/>
    },
    {
        id: 'customsActivities',
        header: "Customs Activities",
        body: <OarTable table={tableContent.customActivities}/>
    },
    {
        id: 'searchDetectionActivities',
        header: "Search & Detection Activities",
        body: <OarTable table={tableContent.searchDetectionActivities}/>
    },
    {
        id: 'cyclamenActivities',
        header: "Cyclamen Activities",
        body: <OarTable table={tableContent.cyclamenActivities}/>
    },
    {
        id: 'otherActivities',
        header: "Other Activities",
        body: <OarTable table={tableContent.OtherActivities}/>
    },
];