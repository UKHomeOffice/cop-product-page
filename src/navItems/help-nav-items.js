export const navItems = [
  {
    name: "Operational Activity Reporting (OAR)", href: "/help/oar/",
    subItems: [{ name: "Overview", href: "/help/oar/" },
      { name: "Submitting your OAR", href: "/help/oar/oar-submitting/" },
      { name: "Checking your OAR Submission", href: "/help/oar/oar-checking/" }]
  },
  {
    name: "Accessing your services", href: "/help/accessing/",
    subItems: [{ name: "Create your COP account", href: "/help/accessing/" }]
  }
  // {
  //   name: "Mandec", href: "/help/mandec/",
  //   subItems: [{ name: "Overview", href: "/help/mandec/" },
  //     { name: "Page-2", href: "/help/mandec/mandec-page-2/" }
  //   ]
  // }
];
