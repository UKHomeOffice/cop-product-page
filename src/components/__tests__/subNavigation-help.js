import React, { useEffect } from "react";
import renderer from "react-test-renderer";
import { navItems } from "../../config/help-nav-items";
import SubNavigation from "../sub_navigation-help";
import { render } from "@testing-library/react";

describe("SubNavigationHelp", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SubNavigation navItems={navItems} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct SubNavigationHelp 'navItems' prop passed in", () => {
    const tree = renderer
      .create(<SubNavigation navItems={navItems} />);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty("navItems", [
      {
        name: "Get started", href: "/help/"
      },
      {
        name: "Operational Activity Reporting (OAR)", href: "/help/oar/",
        subItems: [{ name: "Overview", href: "/help/oar/" },
          { name: "Before you begin", href: "/help/oar/oar-before-you-begin-your-oar/" },
          { name: "Enter shift details and time", href: "/help/oar/oar-entering-shift-details-and-time/" },
          { name: "Understand activity categories", href: "/help/oar/oar-understanding-activity-categories/" },
          { name: "Detection/search activities", href: "/help/oar/oar-detection-search-activities/" },
          { name: "Check your OAR submission", href: "/help/oar/oar-checking-your-oar-submission/" },
          { name: "Add shift debrief notes", href: "/help/oar/oar-add-shift-debrief-notes/" }
        ]
      },
      {
        name: "Mandatory Declarations", href: "/help/mandec/",
        subItems: [{ name: "Overview", href: "/help/mandec/" },
          { name: "Add your line manager", href: "/help/mandec/mandec-adding-your-line-manager/" },
          { name: "If you have something to declare", href: "/help/mandec/mandec-something-to-declare/" },
          { name: "What Line Managers need to do", href: "/help/mandec/mandec-what-line-managers-need-to-do/" }
        ]
      },
      {
        name: "Use of Force", href: "/help/uof/",
        subItems: [{ name: "Overview", href: "/help/uof/" },
          { name: "Before you begin", href: "/help/uof/uof-before-you-begin/" },
          { name: "When you must report use of force", href: "/help/uof/uof-when-you-must-report-use-of-force/" },
          { name: "Personal Safety Training terms", href: "/help/uof/uof-personal-safety-training-terms/" }
        ]
      },
      {
        name: "Events at the Border", href: "/help/eab/",
        subItems: [
          {name: "Overview", href: "/help/eab/"},
          {name: "Events at the Border User Guide", href: "/help/eab/events-at-the-border-user-guide/" },
          {name: "Minimal standard limits", href: "/help/eab/minimal-standard-limits/" },
        ],
      },
      {
        name: "Intelligence Referral", href: "/help/ir/"
      },
      {
        name: "Intelligence Workflow", href: "/help/iw/"
      },
      {
        name: "Accessing COP on mobile devices", href: "/help/accessing-cop-on-mobile/"
      },
      {
        name: "Contact", href: "/help/contact/"
      }
    ]);
  });

  it("Renders correct sub nav list items", () => {
    const navItemsObj = [
      {
        name: "test", href: "/help/test/",
      },
    ];
    const { getByText } = render(<SubNavigation navItems={navItemsObj} />);
    const subNavigationItem = getByText("test");
    expect(subNavigationItem).toBeTruthy();
  });

});