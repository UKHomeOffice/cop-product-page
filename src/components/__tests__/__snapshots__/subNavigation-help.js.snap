// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubNavigationHelp renders correctly 1`] = `
<div
  className="sub-navigation-custom-help"
>
  <nav>
    <ol>
      <div>
         
        <li
          className="sub-navigation__item"
        >
          <a
            href="/help/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Get started
            </span>
          </a>
        </li>
         
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/oar/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Operational Activity Reporting (OAR)
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/mandec/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Mandatory Declarations
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/uof/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Use of Force
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/eab/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Events at the Border
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/ir/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Intelligence Referral
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/iw/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Intelligence Workflow
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/accessing-cop-on-mobile/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Accessing COP on mobile devices
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/using-cop-reports/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Using COP Reports
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/debriefing/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Hot Debrief
            </span>
          </a>
        </li>
      </div>
      <div>
        <li
          className="sub-navigation__item"
        >
          <a
            className="govuk-link subnav"
            href="/help/accessing-cop-on-pc/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Accessing COP on a PC
            </span>
          </a>
        </li>
      </div>
      <div>
         
        <li
          className="sub-navigation__item"
        >
          <a
            href="/help/contact/"
            style={
              Object {
                "color": "black",
              }
            }
          >
            <span>
              Contact
            </span>
          </a>
        </li>
         
      </div>
    </ol>
  </nav>
</div>
`;
